test/k8sT/assertionHelpers.go

```diff
@@ -138,6 +143,9 @@ func DeployCiliumOptionsAndDNS(vm *helpers.Kubectl, ciliumFilename string, optio
 	ExpectCiliumReady(vm)
 	ExpectCiliumOperatorReady(vm)
 	ExpectKubeDNSReady(vm)
+
+	err := vm.WaitforPods(helpers.KubeSystemNamespace, "", longTimeout)
```

Comment:
Shouldn't this block be done only if we are running in GKE?

Reply:
I figured it was harmless anyway. I can put it in an `if` if you'd like